packages/backend/src/routes/index.js

```diff
@@ -28,4 +32,8 @@ router.post('/check/', (req, res, next) => {
   res.send('OK');
 });
 
+router.get('/getCard/:id', async (req, res, next) => {
```

Comment:
We can't have this route. Anyone would be able to get anyone else's credit card

Reply:
Done